.github/workflows/ci_cd.yml

```diff
@@ -127,7 +127,9 @@ jobs:
       - name: Start API, ingest and index test data
         run: just init
       - name: Run API tests
-        run: just api-test -T
+        run: |
+          just api-check -T
```

Comment:
I wonder if we could use the `IS_CI` variable here to automatically add the `-T` flag to the docker args, similar to #569

Reply:
Might be feasible, testing this in deaa5f8.